src/ForgottenPassword/__tests__/ForgottenPassword.test.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import ForgottenPasword from '../index';
```

Comment:
small typo, I see is because it's already  misspelled in `index.js`

Reply:
uh, nice catch